experiment/sync-iso.sh

```diff
@@ -6,6 +6,9 @@ manifest="${PARAM_manifest:-conf/contperf/manifest.zip}"
 inventory="${PARAM_inventory:-conf/contperf/inventory.ini}"
 private_key="${PARAM_private_key:-conf/contperf/id_rsa_perf}"
 
+test_sync_repositories_count="${PARAM_test_sync_repositories_count:-8}"
+test_sync_repositories_url_template="${PARAM_test_sync_repositories_url_template:-http://storage.example.com/iso-repos*}"
+test_sync_repositories_max_sync_secs="${PARAM_test_sync_repositories_max_sync_secs:-600}"
```

Comment:
Maybe rename these to `test_sync_iso_...` (and also in playbook) so it is clear these do not conflict with yum repo sync test playbook. Not a strong option here.

Reply:
I mean - If we decide to run `sync_repositories.yaml` and `sync_iso.yaml` playbooks in one experiment, if vars are named a same way, it will create confusion.

Reply:
changes have been applied to .sh as well as a playbook. So, do we need to change our .doit.sh file also with newly named variables? @jhutar

Reply:
Still `test_sync_repositories_...` where I would expect `test_sync_iso_...`

Reply:
It has been changed. Thank you!